corehq/apps/sms/tests/test_form_session_handler.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime
+
+from django.test import TestCase
+from mock import patch, Mock, MagicMock
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.sms.handlers.form_session import form_session_handler
+from corehq.apps.sms.models import (
+    PhoneNumber,
+    SMS,
+    INCOMING,
+    MessagingEvent,
+    MessagingSubEvent
+)
+from corehq.apps.sms.tests.util import mock_critical_section_for_smsforms_sessions
+from corehq.apps.smsforms.models import SQLXFormsSession
+from corehq.messaging.smsbackends.test.models import SQLTestSMSBackend
+
+
+@patch('corehq.apps.smsforms.util.critical_section_for_smsforms_sessions',
+       new=mock_critical_section_for_smsforms_sessions)
```

Comment:
Could remove the utils and replace them with this:

```suggestion
       new=lambda contact_id: contextlib.supress())
```
More details: https://stackoverflow.com/a/45187287/10840